test/tokens/Xcert.test.js

```diff
@@ -286,12 +289,47 @@ contract('Xcert', (accounts) => {
   });
 
   it('returns the correct NFToken id 2 url', async () => {
-    await xcert.mint(accounts[1], id2, mockProof, 'url2');
+    await xcert.mint(accounts[1], id2, 'url2', proof, config, data);
     const tokenURI = await xcert.tokenURI(id2);
     assert.equal(tokenURI, 'url2');
   });
 
   it('throws when trying to get uri of none existant NFToken id', async () => {
     await assertRevert(xcert.tokenURI(id4));
   });
+
+  it('returns the correct proof for NFToken id', async () => {
```

Comment:
Add a test for missing NFT id

Reply:
Why? It has a modifier validNFToken which has been already tested.

Reply:
Ah, ok, I might have been too quick.